bundle/community/manifests/opentelemetry.io_instrumentations.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.18.0
+    controller-gen.kubebuilder.io/version: v0.17.1
```

Comment:
@TylerHelmuth you should delete your local bin/controller-tools and run `make update`. Unfortunately our tooling doesn't do this automatically.